includes/controllers/class-user.php

```diff
@@ -333,7 +333,12 @@ public function register_user( $request ) {
 		}
 
 		// Register user.
-		$user = ( new Models\User() )->fill(
+		$user = new Models\User();
+
+		// @todo remove temporary fix when updated.
+		$user->set_id( null );
```

Comment:
Please remind why this fix is required

Reply:
Because without this fix model attributes are not saving

Reply:
It is knowm temporary fix which we use in most places. The model does not get all attributes without this temporary fix